lib/bundle/brew_dumper.rb

```diff
@@ -16,12 +16,18 @@ def self.formulae
     end
 
     def self.dump
+      bs = BrewServices.new
       formulae.map do |f|
+        if bs.started?(f[:full_name])
+          restart = ', service_restart: true'
+        else
+          restart = ''
+        end
```

Comment:
Pull the `BrewServices` logic into another method named something like `services_started?`